redisinsight/ui/src/pages/home/HomePage.tsx

```diff
@@ -1,322 +1,58 @@
-import { EuiPage, EuiPageBody, EuiResizableContainer, EuiResizeObserver } from '@elastic/eui'
-import React, { useEffect, useRef, useState } from 'react'
-import { useDispatch, useSelector } from 'react-redux'
-import cx from 'classnames'
-import DatabasePanel from 'uiSrc/pages/home/components/database-panel'
-import { clusterSelector, resetDataRedisCluster, resetInstancesRedisCluster, } from 'uiSrc/slices/instances/cluster'
-import { Nullable, setTitle } from 'uiSrc/utils'
+import { EuiPage, EuiPageBody, EuiResizeObserver, EuiTab, EuiTabs } from '@elastic/eui'
```

Comment:
Maybe will be better do not touch home page. We can get access to rdi pages just from url (Navigation between rdi and databases is not completed, we don't have final design)

Reply:
I have reverted the changes to the `HomePage` component. The latest mocks do seem to use some sort of tabs but we can tackle that in a different card